distributed-map/entry-processor/src/main/resources/hazelcast.xml

```diff
@@ -1,10 +1,16 @@
 <hazelcast xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
            xsi:schemaLocation="http://www.hazelcast.com/schema/config
-                               http://www.hazelcast.com/schema/config/hazelcast-config-3.11.xsd"
+                               http://www.hazelcast.com/schema/config/hazelcast-config-4.0.xsd"
            xmlns="http://www.hazelcast.com/schema/config">
     <network>
         <join>
             <multicast enabled="true"/>
         </join>
     </network>
+
+    <user-code-deployment enabled="true">
```

Comment:
How come this is enabled?

Reply:
I don't even remember why I needed it. Removed in another commit.